chapter_optimization/gd_vn.md

```diff
@@ -380,17 +387,23 @@ Furthermore it may be too expensive to compute via backprop as we would need to
 For now let us ignore such considerations and look at what algorithm we'd get. 
 -->
 
-*dịch đoạn phía trên*
+Để tránh việc kí hiệu vướng víu, ta định nghĩa $H_f := \nabla \nabla^\top f(\mathbf{x})$ là *ma trận Hessian* của $f$.
+Đây là ma trận $d \times d$. Với số $d$ nhỏ và các bài toán đơn giản, ta sẽ dễ tính được $H_f$.
+Nhưng với các mạng sâu, mặc khác, $H_f$ có thể cực lớn, do chi phí lưu trữ $\mathcal{O}(d^2)$ giá trị.
+Hơn nữa nó có thể quá tốn chi phí tính toán thông qua việc lan truyền ngược khi chúng ta muốn áp dụng việc này vào đồ thị gọi lan truyền ngược.
```

Comment:
```suggestion
Hơn nữa nó có thể tốn quá nhiều chi phí tính toán lan truyền ngược.
```
Mình nghĩ thế này đủ nghĩa rồi.

Reply:
"Hơn nữa nó có thể tốn quá nhiều chi phí tính toán khi áp dụng vào lan truyền ngược."
vầy thì sao, sẽ chính xác hơn là chỉ đề cập "chi phí tính toán lan truyền ngược." :v

Reply:
mình không thấy vấn đề gì.
```suggestion
Hơn nữa nó có thể tốn quá nhiều chi phí tính toán khi áp dụng vào lan truyền ngược.
```